### DISCUSSION TITLE
Handling changing town block type

### DISCUSSION THREAD

**Discussion by manul694** [TownyAdvanced/Towny#6016] [Q&A] [answered]
How to handle changing town block type?

**Comment by LlmDl** [comment]
Explain handle.

**Reply by LlmDl** [reply]
That is not a command found in Towny. Are you talking about `/plot set jail`?

**Reply by manul694** [reply]
Yes

**Reply by LlmDl** [reply]
Please explain what you want to know. You need to use more words.

**Comment by ggvbo** [comment, marked as answer]
You could try listening for the `PlotPreChangeTypeEvent` event that you can cancel.
